code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -49,8 +49,8 @@
 //M4A3 PISTOL
 
 /obj/item/weapon/gun/pistol/m4a3
-	name = "\improper M4A3 service pistol"
-	desc = "An M4A3 Colt Service Pistol, the standard issue sidearm of the TerraGov Marine Corps. Uses 9mm pistol rounds."
+	name = "\improper T-27 pistol"
```

Comment:
Keep weapons like this one, which will be resprited, with their old name and description.
The idea is to preserve the old ones and add new ones.